app/routes/admin/events/list.js

```diff
@@ -124,6 +124,11 @@ export default class extends Route.extend(EmberTableRouteMixin) {
       'page[size]'   : params.per_page || 10,
       'page[number]' : params.page || 1
     };
+    // add condition for default sorting the table by date
+    if (!params.sort_by && !params.sort_dir) {
```

Comment:
Now it only works when both parameters are not present. It should work when either of them is absent